wordpress-mobile/WordPress-iOS
# [Stats Refresh] Detail list view: update view

Fixes #11165 

To test:

---
- Go to Insights Comments or Followers.
- Select `View more`.
- Verify the table background is grey.

<img width="350" alt="details_table" src="https://user-images.githubusercontent.com/1816888/53836445-3ffba980-3f4d-11e9-83f1-1bd93520bfc3.png">

---
- On a site that has data for _one_ of the tabs for Insights Comments or Followers, select `View more`
  - Hogwarts > Followers is a good example.
- On the tab with no data, verify the view is:

<img width="350" alt="no_data" src="https://user-images.githubusercontent.com/1816888/53836569-8e10ad00-3f4d-11e9-8845-e8e9aef11585.png">
